### ISSUE TITLE
Applying GradCAM on EfficientNet networks

### ISSUE THREAD

**Issue by francescoalongi** [jacobgil/pytorch-grad-cam#95]
Hello, thank you for providing this implementation of GradCAM.
When I use the GradCAM module with an EfficientNetB4, the `self.activations_and_grads.gradients` vector returned in `base_cam.py:59` results empty, ending up in having a `list index out of range` error in `base_cam.py:74`. I think that a minimal reproducible snippet would be the following:

```python
from efficientnet_pytorch import EfficientNet
model = EfficientNet.from_pretrained('efficientnet-b4')
target_layer = list(model.modules())[-7]

### Everything required for GradCAM here...
```

I think the problem might be in the `target_layer`, but I am not quite sure of that.

**Comment by jacobgil** [comment]
What is the target layer you want to target?
I tried
target_layer = model._conv_head
And it's working well.

**Comment by francescoalongi** [comment]
You are right. I was maybe looking at the wrong Conv2d layer and I was also setting `requires_grad` to `False`.
Thank you for your quick response.
